WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -115,5 +117,22 @@ protected override void EditConfigOnSave(Config config)
 				BitcoinP2PEndPoint = config.GetBitcoinP2pEndPoint().ToString(defaultPort: -1);
 			}
 		}
+
+		public static string GetNodeVersion()
```

Comment:
I would rather return `null` when we can't get a version string (if bitcoind fails to give us result for whatever reason) and that `null` should turn into some user-friendly message for the UI user.